test/functions.test.ts

```diff
@@ -29,4 +29,63 @@ t.test('Functions', async (t) => {
       t.same(data.deep.time, data.topTime)
     })
   })
+
+  t.test('Array namespace', async (t) => {
```

Comment:
Isn't the function already covered by the GROQ test suite?

Reply:
Not atm, and I wanted something to test with locally. Can add some tests to the test suite though!

Reply:
We have tests [here](https://github.com/sanity-io/groq-test-suite/pull/54).

Reply:
Ah! Removed, tested and verified with `GROQTEST_SUITE=../groq-test-suite/suite.ndjson ./test/generate.sh` from the branch